jssource/src_files/include/javascript/EmailsComposeViewModal.js

```diff
@@ -164,14 +164,19 @@
         if (targetCount > 0) {
           targetList = targetList + ',';
         }
-        if (populateModuleName == '') {
+        if (populateModuleName === '') {
           populateModuleName = populateEmailAddress;
         }
         targetList = targetList + populateModuleName + ' <' + populateEmailAddress + '>';
         targetCount++;
       });
       if (targetCount > 0) {
-        $(self.emailComposeView).find('#to_addrs_names').val(targetList);
+        if (populateEmailAddress !== '') {
```

Comment:
What's the point of this? This disables email targets in case the last selected entry doesn't have an email... why?

Reply:
I'm not sure, this was a while back. Might just be an oversight on my part.  
Feel free to submit a PR.

Reply:
ok, thanks :)

Reply:
Thanks to you, nice catch.

Reply:
I've opened a PR: #7760